Autopilot/AttitudeManager/AttitudeManager.c

```diff
@@ -385,8 +381,15 @@ void setHeadingSetpoint(int setpoint){
 void inputCapture(){
     int* channelIn;
     channelIn = getPWMArray(getTime());
-    inputMixing(channelIn, &input_RC_RollRate, &input_RC_PitchRate, &input_RC_Throttle, &input_RC_YawRate, &input_RC_Flap);
-
+    
+    inputMixing(channelIn, &input_RC_RollRate, &input_RC_PitchRate, &input_RC_Throttle, &input_RC_YawRate);
+    
+#if VEHICLE_TYPE == FIXED_WING
```

Comment:
I will try to move all the flap control code to FixedWing.h. Will depend on the PID rewrite I'm working on.